docs/tutorials/how-to-write-contracts-that-talk-to-each-other.md

```diff
@@ -354,7 +354,7 @@ export class CalculatorApi {
 }
 ```
 
-*(For more info on `ContractPromise`, check out [ContractPromise](/docs/runtime-ts/classes/contractpromise) and [ContractPromiseResult](/docs/runtime-ts/classes/contractpromiseresult))*
+*(For more info on `ContractPromise`, check out [ContractPromise](/docs/api/near-sdk-as/classes/_runtime_contract_.contractpromise) and [ContractPromiseResult](/docs/api/near-sdk-as/classes/_runtime_contract_.contractpromiseresult))*
```

Comment:
I'm not sure what to link to here

Reply:
`runtime-ts` is now `near-sdk-as`. these links would go to equivalents in the new auto generated docs folders